viewer/src/components/MoveButton.vue

```diff
@@ -7,11 +7,20 @@
       :booster="button.booster"
       highlighted
     />
+    <svg v-else-if="button.boardAction" width="80" height="80" viewBox="-32 -32 64 64">
```

Comment:
```suggestion
    <svg v-else-if="button.boardAction" width="80" height="80" viewBox="-28 -28 56 56" style="margin: -0.25rem">
```

This removes a lot of the margin as the current margin is too much: 

![image](https://user-images.githubusercontent.com/342922/118038590-9a83c200-b36f-11eb-8c24-7b305908e44d.png)

But to be consistent with the rest of the codebase, when a tile should be able to be displayed directly in the HTML, the `svg` wrapper should be directly put in `BoardAction`, so that you can do:

`<BoardAction v-else-if="button.boardAction" @click="handleClick" :action="button.boardAction" class="mr-1 mb-1" />`

Reply:
you can look at `SpecialAction`, `Booster`, and `TechTile` for examples